third_party/versions.txt

```diff
@@ -15,38 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# Toolchain library versions
-
-GTEST_VERSION=1.8.0
-GFLAGS_VERSION=2.2.0
-GBENCHMARK_VERSION=1.4.1
-
 # Library, Version, Commit Hash
```

Comment:
We are missing versions for bwtree, flatbuffers, libpg_query.

Reply:
Per Matt: bwtree is whatever was in Peloton.
I'll add that we have since modified it: prior to this PR I added instrumentation for Lin's stuff, with this PR I disable ASAN on the BwTree.

Reply:
Yup, +1 on what Matt said.